packages/css/src/utils/string-interpolations.tsx

```diff
@@ -15,43 +11,25 @@ export interface BeforeInterpolation {
 }
 
 /**
- * Will return any suffix out of the CSS and return them both.
+ * Will remove any suffix out of the CSS and return them both.
  *
  * E.g. `'px;font-size: 20px;'` would return `"px"` as the suffix and `";font-size: 20px;"` as the CSS.
```

Comment:
Should we add examples for more cases? Like when regex matched and when not. Might be helpful in future.

Reply:
good idea